tests/jobs/misfit_preprocessor/test_misfit_preprocessor.py

```diff
@@ -223,7 +230,10 @@ def test_misfit_preprocessor_configuration_errors():
 
     config = {
         "unknown_key": [],
-        "clustering": {"spearman_correlation": {"fcluster": {"threshold": 1.0}}},
+        "clustering": {
+            "method": "spearman_correlation",
```

Comment:
So we are changing the schema to require a `method` option. Could we then rename the `spearman_correlation` below to be `clustering_options`? Or does it perhaps make sense to keep it as is..

Looking at this from the user perspective I think it would make sense to use `clustering_options` instead. Not sure about the complexity that would bring for the validation part though, as it is likely easier to create individual schema's with their own validation, rather than doing a "if method x, validate content based on x requirements"

Reply:
We are not adding `method`to the `schema`, we are just changing the default option of `method` from  `spearman_correlation` to `auto_scale`. I do agree that it would be good to change the names of the schema, but was thinking we could have a discussion with the users regarding the names and the schema before doing that.

Reply:
Alright, if this does not introduce any changes to the schema I agree that we leave it as is and do the discussion later. That was what you mentioned on the meeting as well I suppose.

Agreed